src/test/scala/com/joeybaruch/datamodel/AggregatedMetricsSpec.scala

```diff
@@ -0,0 +1,20 @@
+package com.joeybaruch.datamodel
+
+import com.joeybaruch.datamodel.AggregatedMetrics.TruncatedAggregatedMetrics
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class AggregatedMetricsSpec extends AnyFlatSpec with Matchers {
+
+  import AggregatedMetrics._
+
+  behavior of "TruncatedAggregatedMetrics"
+  it should "aggregate correctly" in {
+    truncated1 + truncated2 should equal(truncated1Plus2)
```

Comment:
this test fails because equal on `NamedCountersCollection` is not defined, so it just compares addresses